.github/workflows/test.yml

```diff
@@ -104,151 +104,147 @@ jobs:
         run: cabal build
 
       - name: Set test options
-        # run the tests without parallelism, otherwise tasty will attempt to run
-        # all functional test cases simultaneously which causes way too many hls
-        # instances to be spun up for the poor github actions runner to handle
-        #
         # See https://github.com/ocharles/tasty-rerun/issues/22 for why we need
         # to include 'new' in the filters, since many of our test suites are in the
         # same package.
         run: |
-          echo "TEST_OPTS=-j1 --rerun-update --rerun-filter failures,exceptions,new" >> $GITHUB_ENV
+          echo 'test-options: --rerun-update --rerun-filter "failures,exceptions,new"' >> cabal.project.local
```

Comment:
Nitpick: isn't this what [cabal configure](https://cabal.readthedocs.io/en/stable/cabal-commands.html#cabal-configure) is for?
e.g.
cabal configure --test-options='--rerun-update --rerun-filter failures,exceptions,new'

creates cabal.project.local with the desired options.

Reply:
Good point!